run-valida-tests.sh

```diff
@@ -6,7 +6,7 @@ mkdir ./buildValidaTests
 
 # TODO fix these tests: 2023-11-alloca
 
-TERMTESTS="2023-fibonacci 2023-11-gt-neg 2023-11-lt-neg 2023-12-and-neg 2023-12-div-neg 2023-12-eq-neg 2023-12-ishl-neg 2023-12-ishr-neg 2023-12-log-and-neg 2023-12-log-not-neg 2023-12-mul-neg 2023-12-muli-neg 2023-12-ne-neg 2023-12-not-neg 2023-12-or-neg 2023-12-shl-neg 2023-12-shri-neg 2023-12-shr-neg 2023-12-sra-neg 2023-12-subi-neg 2023-12-sub-neg 2023-12-xor-neg 2023-12-divi-neg 2023-12-divi-signed-neg 2024-1-gti-neg 2024-1-igt-neg 2024-1-lti-neg 2024-1-ilt-neg 2024-1-or-neg 2024-1-address-of-value-on-stack-neg 2024-1-stop 2024-1-null 2024-1-write 2024-3-gte-neg 2024-3-gtei-neg 2024-3-igte-neg 2024-3-lte-neg 2024-3-ltei-neg 2024-3-ilte-neg load_byte storeu8"
+TERMTESTS="2023-fibonacci 2023-11-gt-neg 2023-11-lt-neg 2023-12-and-neg 2023-12-div-neg 2023-12-eq-neg 2023-12-ishl-neg 2023-12-ishr-neg 2023-12-log-and-neg 2023-12-log-not-neg 2023-12-mul-neg 2023-12-muli-neg 2023-12-ne-neg 2023-12-not-neg 2023-12-or-neg 2023-12-shl-neg 2023-12-shri-neg 2023-12-shr-neg 2023-12-sra-neg 2023-12-subi-neg 2023-12-sub-neg 2023-12-xor-neg 2023-12-divi-neg 2023-12-divi-signed-neg 2024-1-gti-neg 2024-1-igt-neg 2024-1-lti-neg 2024-1-ilt-neg 2024-1-or-neg 2024-1-address-of-value-on-stack-neg 2024-1-stop 2024-1-null 2024-1-write 2024-3-gte-neg 2024-3-gtei-neg 2024-3-igte-neg 2024-3-lte-neg 2024-3-ltei-neg 2024-3-ilte-neg load_byte storeu8 2024-4-isge-a 2024-4-isgt-a 2024-4-isle-a 2024-4-sge-a 2024-4-sgei-a 2024-4-sgt-a 2024-4-sgt-b 2024-4-sgti-a 2024-4-sle-a 2024-4-slei-a 2024-4-slt-a 2024-4-slti-a 2024-4-sgt-br 2024-4-sge-b 2024-4-sge-c 2024-4-sge-d 2024-4-sge-e 2024-4-sgt-c 2024-4-sgt-d 2024-4-sgt-e 2024-4-sle-b 2024-4-sle-c 2024-4-sle-d 2024-4-sle-e 2024-4-slt-b 2024-4-slt-c 2024-4-slt-d 2024-4-slt-e"
```

Comment:
Probably can try to use a more generic approach: find all files in the specific folder and return the file name (suffixes stripped):
`find /path/to/folder -type f | awk -F/ '{print $NF}' | awk -F. '{print $1}'`